docs/ossm/ossm2-vs-ossm3.md

```diff
@@ -0,0 +1,103 @@
+
+## The OpenShift Service Mesh 3 operator
+
+OpenShift Service Mesh 3 uses an operator that is maintained upstream as the sail-operator in the istio-ecosystem organization. This operator is smaller in scope and includes significant changes from the operator used in OpenShift Service Mesh 2 that was maintained as part of the Maisra.io project.
+
+## Observability integrations rather than addons
+
+A significant change in OpenShift Service Mesh 3 compared to OpenShift Service Mesh 2 is that the operator no longer installs and manages observability components such as Prometheus and Grafana with the service mesh control plane. It also no longer installs and manages distributed tracing components such as Tempo and OpenTelemetry (Jaeger and Elasticsearch in the past) or the Kiali dashboard. 
+
+The OpenShift Service Mesh 3 operator limits its scope to Istio related resources, with observability components supported and managed by the independent operators that make up [Red Hat OpenShift Observability](https://docs.redhat.com/en/documentation/openshift_container_platform/#Observability) such as [logging](https://docs.redhat.com/en/documentation/openshift_container_platform/#/html/logging/index), user workload [monitoring](https://docs.redhat.com/en/documentation/openshift_container_platform/#/html/monitoring/index), [distributed tracing](https://docs.redhat.com/en/documentation/openshift_container_platform/#/html/distributed_tracing/index). The [Kiali dashboard](https://docs.openshift.com/service-mesh/3.0.0tp1/kiali/ossm-kiali-assembly.html) along with OpenShift Service Mesh Console will continue to be supported with the Kiali operator.
+
+This simplification greatly reduces the footprint and complexity of OpenShift Service Mesh, while providing better, production-grade support for observability through Red Hat OpenShift Observability.
+
+## `Istio` replaces `ServiceMeshControlPlane`
+
+While OpenShift Service Mesh 2 used a resource called `ServiceMeshControlPlane` to configure Istio, OpenShift Service Mesh 3 uses a resource called `Istio`. 
+
+The `Istio` resource contains a `spec.values` field that derives its schema from Istio’s Helm chart values. While this is a different configuration schema than `ServiceMeshControlPlane` uses, the fact that it is derived from Istio’s configuration means that configuration examples from the community Istio documentation can often be applied directly to Red Hat OpenShift Service Mesh’s `Istio` resource. The `spec.values` field has a similar format in `IstioOperator` resource (which is not part of OpenShift Service Mesh), with the `Istio` resource providing an additional validation schema enabling the ability to explore the resource using the OpenShift CLI command `oc explain istios.spec.values`.
```

Comment:
I think here it would make more sense to refer to upstream helm charts (and their values API) rather than the IstioOperator API

Reply:
I did mention helm charts in the first sentence, but I also mentioned IstioOperator as well because it is widely used throughout the Istio.io documentation and has a full API reference in the Istio docs. The only reference I could find for Helm values was in the installation section: https://istio.io/latest/docs/setup/install/helm/#installation-steps, and it seemed a lot more difficult to translate pages like this: https://artifacthub.io/packages/helm/istio-official/istiod?modal=values, into config YAML if you're starting with the Istio. So, at face value, it looked like IstioOperator might be a more familiar starting point for upstream users... or people just reading the docs.

That said, I'm open to how best to convey this... and if the values field in IstioOperator is substantially different, then I should probably remove the reference.